requirements.txt

```diff
@@ -1,6 +1,6 @@
 # TODO set the next point release of openwisp-users when ready
 openwisp-users @ https://github.com/openwisp/openwisp-users/tarball/master
-swapper~=1.1.0
+swapper~=1.2.0
```

Comment:
let's remove this from here since it's already pulled from openwisp-utils (which I just updated)